src/components/ErrorBox.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Radium from 'radium';
+import {connect} from 'react-redux';
+import {styles} from './styles';
+
+export class ErrorBox extends React.Component {
+  render() {
+    switch(false) {
+      case typeof this.props.loadApiError === "undefined":
+        return (
+          <span style={styles.errorBox}>
+              {this.props.loadApiError.message}
+          </span>
+        );
+      case typeof this.props.runCodeError === "undefined":
+        return (
+          <span style={styles.errorBox}>
+              {this.props.runCodeError.message}
+          </span>
+        );
+      case typeof this.props.googleDriveError === "undefined":
+        return (
+          <span style={styles.errorBox}>
+              {this.props.googleDriveError.message}
+          </span>
+        );
+      default:
+        return null; //if no error recorded in state do nothing
+    }
+  }
```

Comment:
The way you are doing this is kind of strange. I would pass in a single `error` prop instead of all three, and then you can just do this:

```javascript
return this.props.error && <span style={styles.errorBox}>{this.props.error.message}</span>;
```